cmd/podman/shared/intermediate_varlink.go

```diff
@@ -0,0 +1,415 @@
+// +build varlink remoteclient
+
+package shared
+
+import (
+	"reflect"
+
+	"github.com/containers/libpod/cmd/podman/varlink"
+	"github.com/sirupsen/logrus"
+)
+
+// ToVarlink converts to the equal varlink structure
+func (c CRStringSlice) ToVarlink() interface{} {
```

Comment:
All of these are exactly the same for all of these, for novarlink and varlink both - can we unify them?

Reply:
I was able to drop the Cr repetitive stuff with a helper func.